composer.json

```diff
@@ -31,6 +31,7 @@
     ]
   },
   "require": {
-    "weberhofer/jsonrpcphp": "^2.2"
+    "weberhofer/jsonrpcphp": "^2.2",
+    "twbs/bootstrap": "3.3.6"
```

Comment:
the dependency is declared in both `composer.json` and `package.json` - let's remove it here.